package.json

```diff
@@ -87,6 +86,7 @@
     "typescript": "4.3.2",
     "use-dark-mode": "^2.3.1",
     "web-vitals": "^1.0.1",
+    "yup": "^0.32.11",
```

Comment:
Do we need both yup and zod? I have no strong feelings about which one we use, but would prefer to stick to one if we can.